src/components/ha-icon-button.ts

```diff
@@ -7,9 +7,21 @@ import "./ha-icon";
 export class HaIconButton extends LitElement {
   @property({ type: Boolean, reflect: true }) disabled = false;
 
-  @property({ type: String }) icon = "";
+  // Regular icon name
+  @property({ type: String }) icon?: string;
 
-  @property({ type: String }) label = "";
+  // SVG icon path
+  @property({ type: String }) path?: string;
+
+  // Label that is used for ARIA support and as default tooltip
+  @property({ type: String }) label?: string;
+
+  @property({ type: Boolean }) showTooltip = true;
```

Comment:
setting is as default `true` doesn't work well, reverse it: `hideTooltip = false`